assets/scss/helpers/_colors.scss

```diff
@@ -20,6 +20,7 @@ $dp-color-lightred: #EC8A83;
 $dp-color-red: #E34F45;
 $dp-color-darkred: #A63A33;
 $dp-color-white: #FFFFFF;
+$dp-color-black: #000000;
```

Comment:
why use the long definition and don't the short?

Reply:
We use the full color variables as there are more complex colors '# A63A33'